Source/WebCore/page/scrolling/ScrollingStateTree.cpp

```diff
@@ -60,6 +60,7 @@ static void nodeWasReattachedRecursive(ScrollingStateNode& node)
 
 ScrollingStateTree::ScrollingStateTree(AsyncScrollingCoordinator* scrollingCoordinator)
     : m_scrollingCoordinator(scrollingCoordinator)
+, m_frameIdentifier(FrameIdentifier { })
```

Comment:
Bad indentation. Can we not get this identifier at ScrollingStateTree construction time? Leaving it unset opens you up to bugs.

Reply:
Fixed. Unfortunately we can't get at the FrameIdentifier from here. We set it after every transaction so I don't think we have to worry about it.